### PR TITLE
Redirect delete

### PR DESCRIPTION
**Pull request by katiecrane**
users can now delete existing redirects

### CODE REVIEW

File: app/Http/Controllers/RedirectsController.php [hunk 1]
```diff
@@ -81,7 +82,13 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        $redirect = Redirect::find($id);
+        var_dump($request->all());
```
Comment (DFurnes) [inline app/Http/Controllers/RedirectsController.php]: :eyes:

Comment (katiecrane) [inline app/Http/Controllers/RedirectsController.php]: 👻

File: app/Http/Controllers/RedirectsController.php [hunk 2]
```diff
@@ -81,7 +82,13 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        $redirect = Redirect::find($id);
```
Comment (DFurnes) [inline app/Http/Controllers/RedirectsController.php]: Not at all necessary for merge, but you should be able to inject this from the controller [with a type-hinted parameter](https://laravel.com/docs/5.2/routing#route-model-binding).

``` diff
-   public function update(Request $request, $id)
+   public function update(Redirect $redirect, Request $request)
   {
-    $redirect = Redirect::find($id);

```

File: config/app.php [hunk 3]
```diff
@@ -157,6 +157,9 @@
         Nightwing\Providers\RouteServiceProvider::class,
 
         Nightwing\Providers\FastlyProvider::class,
+
+        Collective\Html\HtmlServiceProvider::class,
```
Comment (DFurnes) [inline config/app.php:161]: Another thing that's definitely not necessary for merge, but it'd be cool to see if we can get away with not using this dependency for building forms. @weerd's got some nice ideas on how we could handle this with built-in Laravel helpers that you can take a peek at in the Gladiator app.

File: resources/views/redirects/edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <div class='wrapper'>
+        <p>Edit this redirect below. <a href="{{ route('redirects.index') }}">Go back to all redirects.</a></p>
+    </div>
+</div>
+<div class='container__block'>
+    <div class='wrapper'>
+        {!! Form::model($redirect, ['method' => 'PATCH', 'route' => ['redirects.update', $redirect->id]]) !!}
+
+            <div class="form-item">
+                {!! Form::label('path', 'Path: ', array('class' => 'field-label')) !!}
```
Comment (DFurnes) [inline resources/views/redirects/edit.blade.php:16]: `array(...)` → `[...]`

**Comment by DFurnes** [comment]
Looks good! Left some suggestions, but all :+1: from me.

**Comment by DFurnes** [comment]
:+1: MY IMPACT

**Pull request merged**